docs/incubator-api-reference.md

```diff
@@ -16,8 +16,7 @@ KongServiceFacade allows creating separate Kong Services for a single Kubernetes
 Service. It can be used as Kubernetes Ingress' backend (via its path's `backend.resource`
 field). It's designed to enable creating two "virtual" Services in Kong that will point
 to the same Kubernetes Service, but will have different configuration (e.g. different
-set of plugins, different load balancing algorithm, etc.).<br /><br />
-KongServiceFacade requires `kubernetes.io/ingress.class` annotation with a value
+set of plugins, different load balancing algorithm, etc.).<br /><br />KongServiceFacade requires `kubernetes.io/ingress.class` annotation with a value
```

Comment:
Looks like a newline is deleted here. Is it by accident?

Reply:
@randmonkey  This was automatically generated after I upgraded `elastic/crd-ref-docs` to v0.2.0 and executed `make generate`.